### DISCUSSION TITLE
usage with bun js not working

### DISCUSSION THREAD

**Discussion by bobwatcherx** [master-co/css#97] [General]
I tried experimenting with installing it in bun js but it seems it doesn't work well is there a solution for this?

**Comment by 1aron** [comment]
@bobwatcherx What features are you actually using in Bun.js? SSR, AOT or JIT?

**Comment by bobwatcherx** [comment]
Only Bun create react

I wonder if Bun works

**Reply by 1aron** [reply]
We don't have a test or official guide for it yet.

**Comment by itsezc** [comment]
Can confirm Master v2 works with Bun, I have two instances:
- Bun + Next 13 + Webpack
- Bun + Storybook 7 + Vite

Both work for JIT and Storybook works with AOT
